services/buildbot/master/templates/layout.html

```diff
@@ -14,29 +14,44 @@
     {% endif %}
     <title>{{ pageTitle|e }}</title>
     <link rel="stylesheet" href="{{ stylesheet }}" type="text/css" />
-    <link rel="alternate" type="application/rss+xml" title="RSS" href="{{ path_to_root }}rss">
+    <link rel="alternate" type="application/rss+xml" title="RSS" href="{{ path_to_root }}rss" />
+    <link rel="shortcut icon" href="{{ path_to_root }}favicon.ico" />
     {% endblock %}
   </head>
   <body class="interface">
     {% block header -%}
-    <div class="header">
+    <div class="header">    
         <a href="{{ path_to_root or '.' }}">Home</a>
-	- Builds
-	  <a href="{{ path_to_root }}boxes-supported?branch=trunk&num_builds=10">Supported</a>
-	  <a href="{{ path_to_root }}boxes-unsupported?branch=trunk&num_builds=10">Unsupported</a>
-        - <a href="{{ path_to_root }}builders">Builders</a>
-          <a href="{{ path_to_root }}one_line_per_build">Recent Builds</a>
-          <a href="{{ path_to_root }}buildslaves">Buildslaves</a>
+
+        - Builds
+          <a href="{{ path_to_root }}boxes-supported?branch=trunk&num_builds=10">Supported</a>
+          <a href="{{ path_to_root }}boxes-unsupported?branch=trunk&num_builds=10">Unsupported</a>
+
+        {% if authz.advertiseAction('view', request) %}
```

Comment:
I am doing this so that the diff will be minimal ... and I hope that will make future updated easier to follow.

I can remove them if you think that they are a big annoyance